.drone.star

```diff
@@ -853,7 +853,7 @@ def cleanup(config):
 def versionize(version):
   if 'behat_version' in version:
     raw_version = version['behat_version']
```

Comment:
@tboerger is right.

You can use the two different values directly, would be more elegant to not specify "raw_version"